app/src/main/admin/components/ModellingGroups/Models/ModelMetaTable.tsx

```diff
@@ -200,12 +211,24 @@ export class ModelMetaTableComponent extends React.Component<ModelMetaProps, Sta
 }
 
 
-export const mapStateToProps = (state: AdminAppState): ModelMetaProps => {
+export const mapStateToProps = (state: AdminAppState, props: ModelMetaPublicProps): Partial<ModelMetaProps> => {
+
+    const modelExpectations: ILookup<TouchstoneModelExpectations[]> = {};
+    state.groups.models.forEach((model) => {
+        modelExpectations[model.id] = state.groups.expectations
+            .filter(e => e.modelling_group == model.modelling_group && e.disease == model.disease.id);
+    });
 
-    const expectationNotFound  = "Error: expectation not found for model";
+    const includedModels = state.groups.models.filter((model) => {
+        //A model is considered obsolete if it has no expectations
+        return (props.obsoleteModels && modelExpectations[model.id].length == 0) ||
+            (!props.obsoleteModels && modelExpectations[model.id].length > 0)
+    });
 
     return {
-        models: state.groups.models.map(m => {
+        obsoleteModels: props.obsoleteModels,
```

Comment:
This keeps throwing me off as I expect `obsoleteModels` to be an array of obsolete models! Could we make it something more obviously a bool like `obsolete` or `includeObsoleteOnly` or something?